### PR TITLE
show course for student

### PR DESCRIPTION
**Pull request by wowKimsohyun17**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/30243)

## WHAT this PR does?
show course

## Notes Impacted Areas:
- N/A.

## Attachments:
Screenshot from 2020-10-29 10-00-05

### CODE REVIEW

**Comment by wowKimsohyun17** [comment]
ready

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -15,7 +16,16 @@ class HomeController extends Controller
     public function index()
     {
         $courses = Course::with('image')->latest('created_at')->take(config('title.course_number'))->get();
+        $user = $user = Auth::user();
```
Comment (huyenttm-0901) [inline app/Http/Controllers/HomeController.php]: xem lại này

File: app/Http/Controllers/HomeController.php [hunk 2]
```diff
@@ -15,7 +16,16 @@ class HomeController extends Controller
...
         
-        return view('user.component.index', compact('courses'));
+        return view('user.component.index', compact('courses', 'user'));
+    }
+
+    public function course()
+    {
+        $courses = Course::with('image')->get();
+        $user = $user = Auth::user();
```
Comment (huyenttm-0901) [inline app/Http/Controllers/HomeController.php]: tương tự

File: app/Http/Controllers/HomeController.php [hunk 3]
```diff
@@ -15,7 +16,16 @@ class HomeController extends Controller
...
+        
+        return view('user.component.course', compact('courses', 'user'));
```
Comment (congnx-2423) [inline app/Http/Controllers/HomeController.php]: $user = $user = Auth::user()

File: app/Http/Controllers/HomeController.php [hunk 4]
```diff
@@ -15,7 +16,16 @@ class HomeController extends Controller
     public function index()
     {
         $courses = Course::with('image')->latest('created_at')->take(config('title.course_number'))->get();
+        $user = $user = Auth::user();
         
-        return view('user.component.index', compact('courses'));
+        return view('user.component.index', compact('courses', 'user'));
```
Comment (congnx-2423) [inline app/Http/Controllers/HomeController.php]: $user = $user = Auth::user()
$user = auth::user() sao khong dùng @auth bên view

**Comment by wowKimsohyun17** [comment]
ready

**Pull request merged**